reproducer/src/main/java/my/example/otel/reproducer/OpenTelemetryReproducer.java

```diff
@@ -1,8 +1,10 @@
 package my.example.otel.reproducer;
 
+import org.apache.camel.opentelemetry.starter.CamelOpenTelemetry;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+@CamelOpenTelemetry
```

Comment:
That's not required, the application is initializing the Camel OTEL integration using a property (`camel.opentelemetry.enabled`).

Reply:
There was a missing file on the code, I included it (`src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports`).

Reply:
Yes that's right you can in the properties file too, it just needs to be enabled somehow :)